scripts/composer/post-install.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/sh
 
+DOCUMENTROOT=web
```

Comment:
I don't think that there is a good way to pass commandline variables to composer scripts (we expect this script to be run automatically via composer, not directly by the user).  We could change this to:
```
_DOCUMENTROOT=${DOCUMENTROOT-web}
```
Then, you could pass the document root value to the script via an environment variable:
```
DOCUMENTROOT=docroot composer update
```
However, I think this would be a bad idea, as it would cause havoc if you later ran composer update and did not specify the DOCUMENTROOT environment variable.

It would be better, I think, to ask Composer for the directory that `drupal/core` is installed to, and assume that the documet root is the parent directory of that.  Unfortunately, there is no composer command that will divulge that information.  We could potentially add a "tell me the installation location" command to drupal-scaffold, and use that, but it would require another entry in the `scripts` section to make it work.  This would, however, allow us to specify the document root only in composer.json, which would make it easier to customize. Customizing the document root in a committed file is the right way to do it, as you then are not at risk of messing up your update by running a command with the wrong parameters.